### PR TITLE
Fix a bunch more linter items.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: mixer/pkg/runtime/handler/env_test.go [hunk 1]
```diff
@@ -58,16 +58,16 @@ func TestEnv(t *testing.T) {
...
-				t.Errorf("hasStrayWorkers() => %t; wanted %t", got, want)
+			if got := e.(env).hasStrayWorkers(); !got {
+				t.Errorf("hasStrayWorkers() => %t; wanted %t", got, true)
 			}
 			wg.Done()
 		})
 
 		wg.Add(1)
 		e.ScheduleDaemon(func() {
-			if got, want := e.(env).hasStrayWorkers(), true; got != want {
```
Comment (kyessenov) [inline mixer/pkg/runtime/handler/env_test.go]: Why is this style better than original? I certainly prefer having `want` variable.

File: pkg/test/application/echo/client/main.go [hunk 2]
```diff
@@ -95,13 +95,11 @@ func main() {
 	opts, err := newBatchOptions()
 	if err != nil {
 		log.Fatalf("Failed to setup client: %v", err)
-		os.Exit(1)
```
Comment (kyessenov) [inline pkg/test/application/echo/client/main.go]: Does fatal exit these days? I remember the argument was not to exit.

File: mixer/pkg/lang/externs_test.go [hunk 3]
```diff
@@ -431,12 +431,9 @@ func TestExternMatches(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		m, err := externMatches(c.p, c.s)
+		_, err := externMatches(c.p, c.s)
 		if err != nil {
 			t.Fatalf("Unexpected error: %+v, %v", c, err)
-			if m != c.e {
```
Comment (ozevren) [inline mixer/pkg/lang/externs_test.go]: This doesn't seem right?

File: pkg/test/application/echo/client/main.go (hunk 2 above)
Comment (ozevren) [inline pkg/test/application/echo/client/main.go]: I think we should keep these, and avoid using Fatalf.

File: pkg/test/kube/port_forwarder.go [hunk 4]
```diff
@@ -35,7 +35,7 @@ var (
 	// we'l need an index per type.
 	addressMatchIndex = 1
 	forwardRegexIPv4  = regexp.MustCompile(`Forwarding from (127.0.0.1:[0-9]+) -> ([0-9]+)`)
-	forwardRegexIPv6  = regexp.MustCompile(`Forwarding from (\[::1\]:[0-9]+) -> ([0-9]+)`)
+	forwardRegexIPv6  = regexp.MustCompile(`Forwarding from (\[::1]:[0-9]+) -> ([0-9]+)`)
```
Comment (ozevren) [inline pkg/test/kube/port_forwarder.go]: Ugh. How did this work?

File: mixer/pkg/runtime/handler/env_test.go (hunk 1 above)
Comment (geeknoid) [reply mixer/pkg/runtime/handler/env_test.go]: It leads to warnings of redundant variable usage. I improved the code, now I think it's clearly simpler and better than before. PTAL.

File: mixer/pkg/lang/externs_test.go (hunk 3 above)
Comment (geeknoid) [reply mixer/pkg/lang/externs_test.go]: Rejiggered. The original use of t.Fatalf was incorrect, leading to my original refactor. Replacing Fatalf with Errorf makes things make sense now.

File: pkg/test/kube/port_forwarder.go (hunk 4 above)
Comment (geeknoid) [reply pkg/test/kube/port_forwarder.go]: It was a redundant escape, so it was semantically equivalent. My change such removes the redundancy.

File: pkg/test/application/echo/client/main.go (hunk 2 above)
Comment (geeknoid) [reply pkg/test/application/echo/client/main.go]: No, it doesn't exit. GoLand claimed it did and I assumed it was right. I reverted these changes.

**Comment by geeknoid** [comment]
All issues addressed.

PTAL.

**Pull request merged**